.travis.yml

```diff
@@ -1,7 +1,8 @@
 sudo: false
 language: rust
+# MSRV
```

Comment:
I might be misunderstanding the WG post, but should this comment be one line down above the `1.23.0` line?

Reply:
Definitely. Sorry about that. Should be fine now.

Reply:
No worries, thanks for getting this done!